keyboards/boardrun/readme.md

```diff
@@ -0,0 +1,16 @@
+# Boardrun
+
+![Boardrun](https://i.imgur.com/Efo0ItK.jpg)
+
+The Boardrun is a family of ortholinear keyboards with columnar stagger, inspired by the Alice layout. The Classic is a 60% layout that uses Ergodox keycap sets for compatibility, whereas the Bizarre is a 65% layout that uses standard (staggered) keycap sets in an ortholinear arrangement.
```

Comment:
When it states "family", and the layouts being quite different, is this actually 3 separate PCBs?

Reply:
It's one PCB that supports all of these layouts. Pretty cool.

Reply:
this is incorrect. there is a classic pcb and a bizarre pcb.

Reply:
I worked on this same board yesterday in parallel and have some alterations to suggest.  I haven't yet completed work on the Classic but should finish that today fairly quickly.

One major thing is that we really should have two subfolders, "classic" and "bizarre," for the two boards.  Though they are closely related, and shensmobile kept the pin-row-column matrix very similar between them, they are still two very different PCBs with their own sets of layout options.

Reply:
Here's the branch with the changes described:
https://github.com/nopunin10did/qmk_firmware/tree/boardrun

Reply:
#10934 has been created with more complete and accurate implementation of the two Boardrun PCBs.